documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -1155,179 +1155,35 @@ This configuration can be sent via POST to a running Kafka Connect service, whic
 [[monitoring]]
 === Monitoring
 
-Kafka, Zookeeper, and Kafka Connect all have xref:operations/monitoring.adoc[built-in support] for JMX metrics. The Db2 connector also publishes a number of metrics about the connector's activities that can be monitored through JMX. The connector has two types of metrics. Snapshot metrics help you monitor the snapshot activity and are available when the connector is performing a snapshot. Streaming metrics help you monitor the progress and activity while the connector reads CDC table data.
+The {prodname} Db2 connector has three metric types in addition to the built-in support for JMX metrics that Zookeeper, Kafka, and Kafka Connect have.
+
+* <<snapshot-metrics, snapshot metrics>>
+* <<streaming-metrics, streaming metrics>>
+* <<schema-history-metrics, schema history metrics>>
```

Comment:
I kinda liked the explanatory sentences which were there before. How about this:

```suggestion
* <<snapshot-metrics, snapshot metrics>>; for monitoring the connector when performing snapshots
* <<streaming-metrics, streaming metrics>>; for monitoring the connector when reading CDC table data
* <<schema-history-metrics, schema history metrics>>; for monitoring the status of the connector's schema history

Please refer to the xref:operations/monitoring.adoc[monitoring documentation] for details of how to expose these metrics via JMX.
```

Reply:
Change applied and rebased.  Thanks.

Reply:
Changed applied and rebased.